tron/core/action.py

```diff
@@ -117,37 +117,42 @@ def serialize_namedtuple(obj):
         try:
             return json.dumps(
                 {
-                    "command": state_data["command"],
-                    "cpus": state_data["cpus"],
-                    "mem": state_data["mem"],
-                    "disk": state_data["disk"],
-                    "cap_add": state_data["cap_add"],
-                    "cap_drop": state_data["cap_drop"],
+                    "command": state_data.get("command"),
```

Comment:
everything should have a command - sounds like maybe we have some invalid data stored in dynamo?